src/main/resources/config/application.yml

```diff
@@ -190,7 +190,8 @@ application:
     stream-binding-enabled: true
 
     # https://github.com/xm-online/tmf-ms-communication/wiki/Email-Template-Naming-Convention
-    email-path-pattern: \/config\/tenants\/(?<tenantName>[A-Z0-9]*)\/communication\/emails\/(?<templateName>[a-zA-Z0-9-\/]*)\/(?<langKey>[a-z]*)\.ftl
+    email-path-pattern: /config/tenants/{tenantKey}/communication/emails/**/{langKey}.ftl
+    custom-email-path-pattern: /config/tenants/{tenantKey}/communication/customer-emails/**/{langKey}.ftl
```

Comment:
should we name it `/custom-emails` instead of `/customer-emails` ?

Reply:
Changed path to /custom-emails